requirements/build.txt

```diff
@@ -1,4 +1,4 @@
-meson-python>=0.11.0
+git+https://github.com/mesonbuild/meson-python
```

Comment:
I don't think this is enough to make `cibuildwheel` use meson-python from git. The easiest is probably to patch `pyproject.toml`'s `[build-system]` section.